src/pdf.rs

```diff
@@ -0,0 +1,673 @@
+    current_layer.use_text(first_name, 12.0, Mm(60.0), Mm(244.0), &font_light);
+
+    // Last Name
+    current_layer.use_text("Nachname:", 12.0, Mm(21.0), Mm(239.0), &font_light);
+    current_layer.use_text(last_name, 12.0, Mm(60.0), Mm(239.0), &font_light);
+
+    // Email
+    current_layer.use_text("Email-Adresse:", 12.0, Mm(21.0), Mm(234.0), &font_light);
+    current_layer.use_text(
+        email,
+        12.0,
+        Mm(60.0),
+        Mm(234.0),
+        &font_light,
+    );
+
+    // Day on which the pdf was requested
+    let current_date = Local::now().date_naive();
+    let formatted_date = current_date.format("%d.%m.%Y").to_string();
+
+    // Month of the requested 
+    let given_date = NaiveDate::parse_from_str(&format!("{}-01", given_month), "%Y-%m-%d").unwrap();
+    let given_date_year = given_date.year();
+    let given_date_month = given_date.month();
+
+    // German schedaddles
+    let german_months = [
+        "Januar",
+        "Februar",
+        "März",
+        "April",
+        "Mai",
+        "Juni",
+        "Juli",
+        "August",
+        "September",
+        "Oktober",
+        "November",
+        "Dezember",
+    ];
+    let month_name = german_months[(given_date_month - 1) as usize];
+    let formatted_given_month = format!("{} {}", month_name, given_date_year);
+
+    // Month
+    current_layer.use_text("Monat:", 12.0, Mm(148.0), Mm(239.0), &font_light);
+    current_layer.use_text(
+        formatted_given_month,
+        12.0,
+        Mm(168.0),
+        Mm(239.0),
+        &font_light,
+    );
+
+    // Date
+    current_layer.use_text("Datum:", 12.0, Mm(148.0), Mm(234.0), &font_light);
+    current_layer.use_text(formatted_date, 12.0, Mm(168.0), Mm(234.0), &font_light);
+
+    // Schmidt's Handwerksbetrieb
+    current_layer.use_text("Schmidt's", 12.0, Mm(170.0), Mm(275.0), &font_light);
+    current_layer.use_text("Handwerksbetrieb", 12.0, Mm(152.0), Mm(270.0), &font_light);
+
+    // Body-Info Info's
+    let points = vec![
+        (Point::new(Mm(21.0), Mm(150.0)), false),
+        (Point::new(Mm(21.0), Mm(201.0)), false),
+        (Point::new(Mm(188.0), Mm(201.0)), false),
+        (Point::new(Mm(188.0), Mm(150.0)), false),
+    ];
+    let rectangle = Line {
+        points,
+        is_closed: true,
+        has_fill: true,
+        has_stroke: false,
+        is_clipping_path: false,
+    };
+
+    // Body Info Colour
+    let header_colour = Color::Rgb(Rgb::new(0.952, 0.952, 0.952, None));
+    current_layer.set_fill_color(header_colour);
+    current_layer.add_shape(rectangle);
+
+    // Body Info Text
+    let text_colour = Color::Rgb(Rgb::new(0.0, 0.0, 0.0, None));
+    current_layer.set_fill_color(text_colour);
+
+    let used_schedule = get_month_times(given_month, employee_id).await.unwrap();
+    let mut days_in_month = 0;
+
+    // Iterate Days of Month
+    for (_day_num, day_entry) in used_schedule.iter().enumerate() {
```

Comment:
enumerate can be disposed since day_num is dropped, no?

Reply:
"disposed of"*